extensions-contrib/materialized-view-selection/src/main/java/io/druid/query/materializedview/MaterializedViewQueryRunnerFactory.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.materializedview;
+
+import io.druid.query.QueryRunner;
+import io.druid.query.QueryRunnerFactory;
+import io.druid.query.QueryToolChest;
+import io.druid.segment.Segment;
+
+import java.util.concurrent.ExecutorService;
+
+/**
+ * Created by zhangxinyu on 2018/2/5.
+ */
+public class MaterializedViewQueryRunnerFactory implements QueryRunnerFactory 
```

Comment:
Is this class needed in the current implementation?

Reply:
No, it's useless. Should I remove it?

Reply:
Yes please.